docker-compose.maintenance.yml

```diff
@@ -4,7 +4,7 @@ version: '3'
 services:
 
   maintenance:
-    image: nginx:latest
+    image: nginx:stable
```

Comment:
Does this mean a second `nginx` image would need to be pulled when enabling maintenance mode?